polybar-scripts/updates-flatpak/updates-flatpak.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+if ! updates=$(echo "n" | flatpak update 2> /dev/null | wc -l ); then
```

Comment:
We should think about using `flatpak update --noninteractive`? This might be cleaner.

Reply:
I tried using that and the problem is it just runs the update. `flatpak update --help` doesn't list any way to only print updates.

Reply:
Mh k, that's annoying. I had tested it, but I have not tried it with updates.